roles/v2/pool/src/lib/mining_pool/message_handler.rs

```diff
@@ -104,9 +106,9 @@ impl ParseDownstreamMiningMessages<(), NullDownstreamMiningSelector, NoRouting>
                         version: share.get_version(),
                         header_timestamp: share.get_n_time(),
                         header_nonce: share.get_nonce(),
-                        coinbase_tx: coinbase.try_into().unwrap(),
+                        coinbase_tx: coinbase.try_into()?,
                     };
-                    // TODO we can block everything with the below
+                    // TODO we can block everything with the below (looks like this will infinite loop??)
```

Comment:
Yeah - looks like it'll just spam the `solution_sender` if that try_send fails... @Fi3 said he's implementing this branch now so hopefully it'll resolve itself soon.